content/source/docs/enterprise-beta/api/policies.html.md

```diff
@@ -0,0 +1,256 @@
+
+| Method | Path           |
+| :----- | :------------- |
+| POST | /organizations/:organization/policies |
+
+### Parameters
+
+- `:organization` (`string: <required>`) - specifies the organization name where the Policy will be defined
+- `name` (`string: <required>`) - specifies the name of the policy. Accepts alphanumeric characters, as well as `-` and `_`. Cannot be modified after creation.
+- `mode` (`string: "soft"`) - specifies the enforcement level of the policy, it can be `hard`, `soft`, or `advisory`. Enforcement level details can be found in the [Managing Policies](/docs/enterprise-beta/sentinel/manage-policies.html) documentation.
+
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "attributes": {
+      "enforce": [
+        {
+          "path": "my-example-policy.sentinel",
+          "mode": "hard"
+        }
+      ],
+      "name": "my-example-policy",
+    },
+    "type":"policies"
+  }
+}
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://atlas.hashicorp.com/api/v2/organizations/my-organization/policies
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id":"pol-u3S5p2Uwk21keu1s",
+    "type":"policies",
+    "attributes": {
+      "name":"basic",
+      "enforce": [
+        {
+          "path":"basic.sentinel",
+          "mode":"advisory"
+        }
+      ],
+      "updated-at":"2017-10-10T20:52:13.898Z"
+    },
+    "links": {
+      "self":"/api/v2/policies/pol-u3S5p2Uwk21keu1s",
+      "upload":"/api/v2/policies/pol-u3S5p2Uwk21keu1s/upload",
+      "download":"/api/v2/policies/pol-u3S5p2Uwk21keu1s/download"
+    }
+  }
+}
+```
+
+## Upload a Policy
+
+-> **Note**: This endpoint is not JSON-API enabled therefore the serialization format of the body and the HTTP headers do not follow the JSON-API patterns of other endpoints.
+
+This endpoint is used to upload the policy file to the policy.
+
+
+| Method | Path           |
+| :----- | :------------- |
+| PUT | /organizations/:organization/policies/:policy_id/upload |
+
+### Parameters
+
+- `:organization` (`string: <required>`) - specifies the organization name where the Policy will be defined
+- `:policy_id` (`string: <required>`) - specifies the policy ID to which the file should be saved
+
+### Sample Payload
+
+```plain
+main = rule { true }
+```
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/octet-stream" \
+  --header "Accept: text/plain" \
+  --request POST \
+  --data @payload.json \
+  https://atlas.hashicorp.com/api/v2/organizations/my-organization/policy/pol-u3S5p2Uwk21keu1s/upload
+```
+
+
+## Update a Policy
+
+This endpoint enables you to update a Policy
+
+| Method | Path           |
+| :----- | :------------- |
+| PATCH | /organizations/:organization/policies/:policy_id |
```

Comment:
This is not nested under org. I would just take the self link from the links sections above.